From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: guihkx <626206+guihkx@users.noreply.github.com>
Date: Fri, 18 Oct 2024 05:06:18 -0300
Subject: Save lyrics to $XDG_DATA_HOME/lyrics

---
 src/Application.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Application.vala b/src/Application.vala
index e35e54d..342222d 100644
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -1,6 +1,6 @@
 
 public class Lyrics.Application : Gtk.Application {
-    public static string DEFAULT_LYRICS_DIR = Environment.get_home_dir ()+"/.lyrics/";
+    public static string DEFAULT_LYRICS_DIR = Environment.get_user_data_dir () + "/lyrics/";
     public static GLib.Settings settings = new Settings ("com.github.naaando.lyrics");
     public LyricsService lyrics_service { get; set; }
 
-- 
2.47.0

