--- a/src/Application.vala	2019-03-17 03:17:41.000000000 +0100
+++ b/src/Application.vala	2019-10-11 21:54:49.258301739 +0200
@@ -35,6 +35,10 @@
         players.on_active_player_changed.connect (() => display_view.on_player_change (players.active_player));
 
         var main_window = new MainWindow (this, players, main_stack);
+         
+        Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+        Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
+
         var header_bar = new Lyrics.HeaderBar (players, lyrics_service);
         main_window.state_flags_changed.connect (() => {
             header_bar.parent_window_is_active = !(Gtk.StateFlags.BACKDROP in main_window.get_state_flags ());
